beacon_node/client/src/notifier.rs

```diff
@@ -63,7 +72,7 @@ pub fn spawn_notifier<T: BeaconChainTypes>(
                     );
                     eth1_logging(&beacon_chain, &log);
                     bellatrix_readiness_logging(Slot::new(0), &beacon_chain, &log).await;
-                    capella_readiness_logging(Slot::new(0), &beacon_chain, &log).await;
+                    post_capella_readiness_logging(Slot::new(0), &beacon_chain, &log).await;
```

Comment:
All forks after capella forgot to add the notifier here, resulting is the lack of readiness checks pre-genesis. Since it only affects devnets It was not noticed probably